app/itemsService.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  var app = angular.module('toDoList');
+
+  var items = [];
+  var id = 0;
+
+  app.service('itemsApi', function() {
+    this.items = items;
+
+    this.addItem = function(item) {
+      if(item.description) {
+        item.id = id++;
+        item.done = false;
+        this.items.push(item);
+      }
+    };
+
+    this.deleteItem = function(item) {
+      items.forEach(function(element, index) {
+        if(element.id == item.id) {
```

Comment:
Not required but I recommend you to do a small read on triple-equals operator in JavaScript.